pkg/server/backend_manager.go

```diff
@@ -18,15 +18,43 @@ package server
 
 import (
 	"context"
+	"fmt"
 	"math/rand"
+	"strings"
 	"sync"
 	"time"
 
 	"k8s.io/klog/v2"
 	client "sigs.k8s.io/apiserver-network-proxy/konnectivity-client/proto/client"
+	pkgagent "sigs.k8s.io/apiserver-network-proxy/pkg/agent"
 	"sigs.k8s.io/apiserver-network-proxy/proto/agent"
 )
 
+type ProxyStrategy string
+
+const (
+	ProxyStrategyDefault  ProxyStrategy = "default"
+	ProxyStrategyDestHost ProxyStrategy = "destHost"
```

Comment:
Not blocking but we should really document what each proxy strategy means. Eg.
"default": With this strategy the Proxy Server will randomly pick a backend from the current healthy backends to establish the tunnel overwhich to forward requests.

Reply:
add a description for each proxy strategy